lib/index.ts

```diff
@@ -131,6 +130,44 @@ class LogService {
     }
   }
 
+  addLogger(
+    loggerScopeFunction: (config: InitialConfig) => LoggerPlugin,
+    pluginConfig: PluginConfig) {
+    if (typeof loggerScopeFunction !== 'function')
+      return;
+
+    const loggerOptions = loggerScopeFunction.call(this, this.globalConfig);
+
+    if (loggerOptions == null)
+      return;
+
+    const loggerKeys = Object.keys(loggerOptions);
+    const requiredKeys = ['error', 'warn', 'info', 'debug'];
```

Comment:
Use `LogLevel` enum to refer to this methods names.

Reply:
Come again, I did not understand!

Reply:
AHmn, got it now!!